next-docs/pages/world/system-hooks.mdx

```diff
@@ -1 +1,14 @@
 # System hooks
+
+The [namespace owner](/world/namespaces-access-control#ownership) can [register](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol#L65-L81) hooks that are called before any call to a specific `System`, after such a call, or both before and after the call.
+
```

Comment:
A code example for registering system hooks would be useful here, but fine to do as a follow up

Reply:
Added to the Notion doc